float64 x
float64 y
float64 z
float64 v0
float64 v1
float64 v2
---

